localization/english/event_localization/wc_draenor_events_l_english.yml

```
@@ -0,0 +1,21 @@
+﻿l_english:
+ WCDRA_5_TITLE:0 "The Dark Portal"
+ 
+ WCDRA_3_DESC:0 "Your army has reached the Dark Portal. After the long siege, your troops have occupied the surrounding area and are now awaiting your orders.\n\nFel energy forms in the portal's center, creating a window into the other world. Enemies can step through at any moment so you have to make a decision quickly: put an end to this accursed gateway once and for all or claim it for your own purposes."
```

Made two small changes. The second one is not strictly necessary, but the `for your own purposes` never sounded right to me, and I think this is better.
```suggestion
 WCDRA_3_DESC:0 "Your army has reached the Dark Portal. After the long siege, your troops have occupied the surrounding area and are now awaiting your orders.\n\nFel energy forms in the portal's center, creating a window into the other world. Enemies can step through at any moment so you must make a decision quickly: put an end to this accursed gateway once and for all or claim it for yourself."
```